queries/generators/switzerland.rq

```diff
@@ -0,0 +1,29 @@
+# expected_result_count: 220
+SELECT DISTINCT
+  ?qid
+  ?orgLabel
+  ?orgDescription
+  ?type
+  ?typeLabel
+  ?country
+WHERE {
+  BIND(wd:Q39 AS ?country)
+
+  VALUES ?type {
+    wd:Q113025511 # government agency of switzerland (72)
+    wd:Q113015553 # federal ministry of switzerland (6)
```

Comment:
Should be 6 or 7? [On-wiki 7 is mentioned](https://www.wikidata.org/wiki/Wikidata:WikiProject_Govdirectory/Switzerland#Federal_Council). I think it would be best if you find a source to back either of them up.

Reply:
@Ainali There should be 7 of them. But this is not the only issue. 

1. government agency of switzerland (72) [https://w.wiki/865d](https://w.wiki/865d) (70)
2. canton authorities (116) [https://w.wiki/865i](https://w.wiki/865i) (213)

Reply:
It would be great if we could find a source for the agencies so that we can compare which ones are missing.